README.md

```diff
@@ -534,6 +534,18 @@ await $`pwd`
 EOF
 ```
 
+#### Attaching .bash_profile/.zshrc
+
+By default `child_process` does not include aliases and bash functions (https://github.com/nodejs/node/issues/3916). 
+But you are still able to do it by hand. Just attach necessary directives to `$.prefix`.
+
+```js
+{
+  $.prefix = 'export NVM_DIR=$HOME/.nvm; source $NVM_DIR/nvm.sh; set -e;'
```

Comment:
People will drop pipefall. Maybe better to suggest $.prefix += ...

Reply:
Reasonable.